team-b-api/src/main/resources/application.yml

```
@@ -1 +1,15 @@
+spring:
+  datasource:
+    driver-class-name: com.mysql.cj.jdbc.Driver
+    url: ${DB_URL}
+    username: ${DB_USERNAME}
+    password: ${DB_PASSWORD}
 
+  jpa:
+    open-in-view: false
+    hibernate:
+      ddl-auto: update
+    show-sql: true
```

tip; 실무에서는 JPA 로 DDL 을 실행시키는 경우는 거의 없어요
보통 DB에 DDL 을 직접 수행하고 JPA 는 Query 용도로만 사용한답니다
```suggestion
    hibernate:
      ddl-auto: none
    show-sql: true
```

---

production 환경에서는 반드시 none으로 해야하는것은 알고있었는데 혹시 개발하는 환경에서도 none으로 해놓고 DB에 직접 DDL을 수행하는 편인가요?